lib/src/main/kotlin/com/swmansion/starknet/data/types/transactions/Transaction.kt

```diff
@@ -33,6 +33,15 @@ enum class TransactionType(val txPrefix: Felt) {
     L1_HANDLER(Felt.fromHex("0x6c315f68616e646c6572")), // encodeShortString('l1_handler')
 }
 
+@Serializable
+enum class DAMode {
```

Comment:
It's `DA_MODE` in the spec, but naming this `DAMode` feels very non-java, so maybe we should deviate from the spec a bit and rename it to `DataAvailabilityMode`?

Reply:
Let's leave it as is